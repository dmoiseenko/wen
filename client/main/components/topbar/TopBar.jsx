import React from 'react';

import NavBarContainer from './nav/NavBar.container';
import NavToggleContainer from './nav/NavToggle.container';
import NavLogo from './nav/NavLogo';


function TopBar() {
  return (
    <div className="container">
      <NavLogo />
      <NavToggleContainer />
      <NavBarContainer />
    </div>
  );
}

export default TopBar;
