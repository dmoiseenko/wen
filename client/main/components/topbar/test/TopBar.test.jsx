import React from 'react';
import { shallow } from 'enzyme';

import TopBar from '../TopBar';


it('should render', () => {
  const wrapper = shallow(
    <TopBar />
  );

  expect(wrapper.node).toMatchSnapshot();
});
