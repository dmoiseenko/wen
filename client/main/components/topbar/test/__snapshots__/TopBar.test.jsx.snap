// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render 1`] = `
<div
  className="container"
>
  <NavLogo />
  <Connect(NavToggle) />
  <withRouter(Connect(NavBar)) />
</div>
`;
